risc0/zkvm/src/host/api/client.rs

```diff
@@ -666,7 +665,7 @@ impl Client {
                                     .try_into()?,
                                 }),
                                 None => Err(malformed_err()),
-                            }
+                            };
```

Comment:
would be nice to revert changes to this file, to make backporting easier down the road